modules/services/hound.nix

```diff
@@ -41,7 +43,7 @@ in {
     };
 
     repositories = mkOption {
-      type = types.attrsOf (types.uniq types.attrs);
+      type = types.attrsOf (types.uniq (types.attrsOf types.anything));
```

Comment:
this should actually be `jsonFormat.type` instead of `types.attrsOf types.anything`

Reply:
I pushed a new version with simply `types.attrsOf jsonFormat.type`, which I think should be correct.